esx_service/utils/vmdk_utils.py

```diff
@@ -94,7 +94,7 @@ def vmdk_is_a_descriptor(filepath):
                 line = f.readline()
                 return line.startswith('# Disk DescriptorFile')
         except:
-            logging.warning("Failed to open %s for descriptor check", filepath)
+            logging.exception("Failed to open %s for descriptor check", filepath)
```

Comment:
I've added this specific one in PR description. What do you mean by catch generic exception? "except:" will cause any exception in this case rt?